realtime/realtime-web/src/main/java/com/navercorp/pinpoint/realtime/web/atc/websocket/RedisActiveThreadCountHandler.java

```diff
@@ -0,0 +1,53 @@
+package com.navercorp.pinpoint.realtime.web.atc.websocket;
+
+import com.navercorp.pinpoint.realtime.web.atc.service.DemandRegisterService;
+import com.navercorp.pinpoint.web.websocket.ActiveThreadCountHandler;
+import com.navercorp.pinpoint.web.websocket.PinpointWebSocketHandler;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.lang.NonNull;
+import org.springframework.web.socket.CloseStatus;
+import org.springframework.web.socket.WebSocketSession;
+
+import java.util.Objects;
+
+public class RedisActiveThreadCountHandler extends ActiveThreadCountHandler implements PinpointWebSocketHandler {
+
+    private static final Logger logger = LogManager.getLogger(RedisActiveThreadCountHandler.class);
+
+    private final DemandRegisterService demandRegisterService;
+
+    public RedisActiveThreadCountHandler(DemandRegisterService demandRegisterService) {
+        super(null);
+        this.demandRegisterService = Objects.requireNonNull(demandRegisterService, "demandRegisterService");
+    }
+
+    @Override
+    public void afterConnectionEstablished(@NonNull WebSocketSession session) {
+        logger.info("ATC Connection Established. session: {}", session);
+        this.demandRegisterService.registerSession(session);
+    }
+
+    @Override
+    public void afterConnectionClosed(@NonNull WebSocketSession session, @NonNull CloseStatus status) {
+        logger.info("ATC Connection Closed. session: {}, status: {}", session, status);
+        this.demandRegisterService.unregisterSession(session);
+    }
+
+    @Override
+    public void start() {}
+
+    @Override
+    public void stop() {}
+
+    @Override
+    public int getPriority() {
+        return 1;
```

Comment:
Currently, the realtime function is turned on/off by this function: `getPriority`
How about separate the API and turned on/off this realtime feature with experimental config? (`config/experimental`)
Cause the existing implementation for the realtime feature does not removed by this PR, The web may? control its feature by added configuration.

Or you can just pull out this priority `.properties`.

Reply:
Providing safety option is internally discussed, but postponed because of the cost issue